README.md

```diff
@@ -33,14 +33,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
-
+Semantic HTML gives the content meaning.
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
-
+Block refers to block level elements and inline refers to line level elements
```

Comment:
Expand, check this out: https://stackoverflow.com/questions/9189810/css-display-inline-vs-inline-block